src/components/Frame/Frame.scss

```diff
@@ -28,10 +28,14 @@ $skip-vertical-offset: rem(10px);
   transform: translateX(0%);
 
   @include frame-when-nav-displayed {
-    z-index: 1;
-    top: top-bar-height();
+    z-index: z-index(nav, $fixed-element-stacking-order);
```

Comment:
Can you give me some more context on all these z-index changes? The z-index of the app frame is a bit of a delicate dance, so these changes give me concerns. It'd be great to have annotations around what each change to `z-index` does.

This style is applied above already (line 20), so I wonder if/why this is even needed.